content/v2.0/monitor-alert/send-email.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Send alert email
+description: >
+  Send an alert email.
+menu:
+  v2_0:
+    parent: Monitor & alert
+weight: 104
+v2.0/tags: [alert, email, notifications, check]
+related:
+  - /v2.0/monitor-alert/checks/
+---
```

Comment:
Doesn't need to be changed in this PR, but when you migrate this to the `v1-merge` branch use this frontmatter:

```
---
title: Send alert email
description: >
  Send an alert email.
menu:
  influxdb_2_0:
    parent: Monitor & alert
weight: 104
influxdb/v2.0/tags: [alert, email, notifications, check]
related:
  - /influxdb/v2.0/monitor-alert/checks/
---
```

Reply:
Thank you! Saving snippet for monitoring PRs....